packages/acceptance/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "acceptance",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@hig/button": "^1.0.0",
```

Comment:
after today's (12/11) release `button`, `flyout`, and `theme-data` got version bumps

Reply:
Thanks. Updated on the branch.